modules/tinymce/src/core/main/ts/api/util/XHR.ts

```diff
@@ -59,10 +59,152 @@ const XHR: XHR = {
 
   /**
    * Sends a XMLHTTPRequest.
-   * Consult the TinyMCE Documentation for details on what settings this method takes.
    *
    * @method send
-   * @param {Object} settings Object will target URL, callbacks and other info needed to make the request.
+   * @param {Object} settings Object will target URL, callbacks and other info needed to make the request.<br />
```

Comment:
This generates rather odd/unexpected HTML... Are we sure we want this as a child of the list item? eg:
```
<li>
   - Object will target URL, callbacks and other info needed to make the request.
   <div>
   ...table
   </div>
</li>
```

Reply:
I'm not sure if it's useful, but moxiedoc does support the `@setting` markup as well. That'll generate a table at the top of the API docs with the names.

Reply:
added another br element to improve layout of output docs in https://github.com/tinymce/tinymce/pull/5924/commits/57feb07c8ca8a429341dfdff3c37241249bfab1b